api/src/main/java/org/pmiops/workbench/db/model/DbInstitutionTierRequirement.java

```diff
@@ -0,0 +1,110 @@
+package org.pmiops.workbench.db.model;
+
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "institution_tier_requirement")
+public class DbInstitutionTierRequirement {
+  public enum RequirementEnum {
```

Comment:
nit: same as other comment, would probably not call this `Enum` unless there's no other option